vision/style_transfer/README.md

```diff
@@ -0,0 +1,82 @@
+|Output Image      |[<img src="images/output_images/amber_mosaic.jpg">](images/style_images/amber_mosaic.jpg)| [<img src="images/output_images/amber_candy.jpg">](images/style_images/amber_candy.jpg)|[<img src="images/output_images/amber_rain_princess.jpg">](images/style_images/amber_rain_princess.jpg)| [<img src="images/output_images/amber_udnie.jpg">](images/style_images/amber_udnie.jpg)|[<img src="images/output_images/amber_pointilism.jpg">](images/style_images/amber_pointilism.jpg)|
+
+
+## Description
+The model uses the method described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/pdf/1607.08022.pdf).
+  
+
+## Model
+ |Model        |Download  |MD5 Checksum| Download (with sample test data)|ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|Mosaic|[6,571 KB](models/mosaic.onnx)  | 39f0d4d12cf758a7aa31eb150d66244a| [16,703 KB](models/transform_mosaic.tar.gz)|1.5.0|9|
+|Candy|[6,571 KB](models/candy.onnx)  | ef6b9b26d2821ee0c082f229b2e6efcd| [16,768 KB](models/transform_candy.tar.gz)|1.5.0|9|
+|Rain Princess|[6,571 KB](models/rain_princess.onnx)  | 8253cf9670bb24b38152bd71de5571f1|[16,829 KB](models/transform_rain_princess.tar.gz)|1.5.0|9|
+|Udnie|[6,571 KB](models/udnie.onnx)  | f3797cf0dd731c83b307ffa76aed2e67| [16,765 KB](models/transform_udnie.tar.gz)|1.5.0|9|
+|Pointilism|[6,571 KB](models/pointilism.onnx)  | e3241660ecd9f14a671d7229bf18cbd1| [16,741 KB](models/transform_pointilism.tar.gz)|1.5.0|9|
+<hr>
+
+## Inference
+Refer to [fns-pytorch-to-onnx.ipynb](fns-pytorch-to-onnx.ipynb) for detailed preprocessing and postprocessing.
+
+### Input to model
+The input to the model are 3-channel RGB images. The images have to be loaded in a range of [0, 255]. If running into memory issues, try resizing the image by increasing the scale number. 
+
+### Preprocessing steps
+```
+from PIL import Image
+import numpy as np
+
+# loading input and resize if needed
+image = Image.open("PATH TO IMAGE")
+scale = 1 # increase the scale when running into memory issues
+image = image.resize((int(image.size[0] / scale), int(image.size[1] / scale)), Image.ANTIALIAS)
+
+# Preprocess image
+x = np.array(image).astype('float32')
+x = np.transpose(x, [2, 0, 1])
+x = np.expand_dims(x, axis=0)
+```
+
+### Output of model
+The converted ONNX model outputs a NumPy float32 array of shape [10, 3, ‘height’, ‘width’). The height and width of the output image are the same as the height and width of the input image. 
```

Comment:
The batch size is 1. However, when I visualized the ONNX model in Netron, it gave me an input shape of [10,3,H,W].

Reply:
@spandantiwari do you know why it would be that way?

Reply:
The input shape comes from the input that was used in the call to the torch.onnx.export API. @jennifererwangg Could you check what is the shape of the dummy input you use for the API call? Just as an aside - ORT will run any batch size so long as it is less than the encoded shape.

Reply:
I set the shape of the dummy input to (10, 3, 244, 244) when I made the API call. Should I change the README to "shape of [1, 3, 'height', width']" instead?

Reply:
OK, that's expected. We should set the dummy input shape to `[1, 3, 224, 224]` at the very least. It would be better to make the batch size dimension variable length, i.e. we could send input of any batch size. We can do this by using the `dynamic_axes` argument in `torch.onnx.export` API. Let me know if there are any questions.